runing/routers.py

```
@@ -0,0 +1,51 @@
+@router.get("/{user_id}", response_model=schemas.User)
+def read_user(
+    user_id: int, 
+    user_service: UserService = Depends(get_user_service)
+):
+    return user_service.get_user(user_id)
+
+@router.put("/{user_id}", responce_model=schemas.User)
+def update_user(
+    user_ud: int,
+    user_update: schemas.UserUpdate,
+    user_service: UserService = Depends(get_user_service)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)